_integration-test/run.py

```diff
@@ -0,0 +1,159 @@
+import subprocess
+import os
+from functools import lru_cache
+import requests
+import unittest
+import sentry_sdk
+import time
+import json
+
+SENTRY_CONFIG_PY = "sentry/sentry.conf.py"
+SENTRY_TEST_HOST = os.getenv("SENTRY_TEST_HOST", "http://localhost:9000")
+TEST_USER = "test@example.com"
+TEST_PASS = "test123TEST"
+TIMEOUT_SECONDS = 60
+
+
+def run_shell_command(cmd, **kwargs) -> subprocess.CompletedProcess:
+    return subprocess.run(cmd, shell=True, check=True, **kwargs)
+
+
+class IntegrationTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.session = requests.Session()
+        response = None
+        run_shell_command("docker compose --ansi never up -d")
+        for i in range(TIMEOUT_SECONDS):
```

Comment:
this is pretty ugly, open to suggestions on how to make polling for a response cleaner